packages/react-charting/src/components/StackedBarChart/MultiStackedBarChartRTL.test.tsx

```diff
@@ -0,0 +1,327 @@
+    yAxisCalloutData: '35%',
+  },
+  {
+    legend: 'Credit card numbers',
+    data: 92,
+    color: DefaultPalette.blueLight,
+    xAxisCalloutData: '2020/04/30',
+    yAxisCalloutData: '92%',
+  },
+  {
+    legend: 'Tax identification numbers (USA)',
+    data: 87,
+    color: DefaultPalette.black,
+    xAxisCalloutData: '2020/04/30',
+    yAxisCalloutData: '87%',
+  },
+];
+
+const secondChartPoints: IChartDataPoint[] = [
+  {
+    legend: 'Phone Numbers',
+    data: 40,
+    color: DefaultPalette.blue,
+    xAxisCalloutData: '2020/04/30',
+    yAxisCalloutData: '87%',
+  },
+  {
+    legend: 'Credit card Numbers',
+    data: 23,
+    color: DefaultPalette.green,
+    xAxisCalloutData: '2020/04/30',
+    yAxisCalloutData: '87%',
+  },
+];
+
+const data: IChartProps[] = [
+  {
+    chartTitle: 'Monitored',
+    chartData: firstChartPoints,
+  },
+  {
+    chartTitle: 'Monitored Second Chart',
+    chartData: firstChartPoints1,
+  },
+  {
+    chartTitle: 'Unmonitored',
+    chartData: secondChartPoints,
+  },
+];
+
+describe('Multi Stacked bar chart rendering', () => {
+  testWithoutWait(
+    'Should render the Multi Stacked bar chart with non-empty data',
+    MultiStackedBarChart,
+    { data },
+    container => {
+      // Assert
+      expect(container).toMatchSnapshot();
+    },
+  );
+});
+
+describe('Multi Stacked bar chart - Subcomponent bar', () => {
+  testWithWait(
+    'Should render the multi stacked bar with the given bar height',
+    MultiStackedBarChart,
+    { data, barHeight: 5 },
+    container => {
+      // Assert
+      expect(getById(container, /_MSBC_bar/i)).toHaveLength(data.length);
+      expect(getById(container, /_MSBC_rect/i)[0].getAttribute('height')).toEqual('5');
+    },
+  );
+
+  testWithWait(
+    'Should render the multi stacked bar with the given variant part-to-whole',
+    MultiStackedBarChart,
+    { data, variant: 'part-to-whole' },
+    container => {
+      const rect = getById(container, /_MSBC_rect/i);
+      // Assert
+      expect(getById(container, /_MSBC_bar/i)).toHaveLength(data.length);
+      expect(getById(container, /_MSBC_rect/i)).toHaveLength(12);
+      const rect1: number = +rect[10].getAttribute('width')!.split('%')[0];
+      const rect2: number = +rect[11].getAttribute('width')!.split('%')[0];
+      expect(rect1 + rect2).toEqual(100);
+    },
+  );
+
+  testWithWait(
+    'Should render the multi stacked bar with the given variant part-to-whole',
+    MultiStackedBarChart,
+    { data, variant: 'absolute-scale' },
+    container => {
+      const rect = getById(container, /_MSBC_rect/i);
+      // Assert
+      expect(getById(container, /_MSBC_bar/i)).toHaveLength(data.length);
+      expect(getById(container, /_MSBC_rect/i)).toHaveLength(12);
+      const rect1: number = +rect[10].getAttribute('width')!.split('%')[0];
+      const rect2: number = +rect[11].getAttribute('width')!.split('%')[0];
+      expect(rect1 + rect2).not.toEqual(100);
```

Comment:
We can make the test more robust. Here the biggest data point group has 100% width and other groups have width as per the values.

Reply:
done